src/transactions/transaction/transactionBuilder.ts

```diff
@@ -61,7 +61,7 @@ export default class TransactionBuilder {
     return this;
   }
 
-  withFamilyVersion(self, familyVersion: string): TransactionBuilder {
+  withFamilyVersion(familyVersion: string): TransactionBuilder {
```

Comment:
Shouldn't the TypeScript compiler caught this?

Reply:
I thought so but I guess not

Reply:
That seems like that should be a build error.  Perhaps this can be followed up in a PR, after the issue is investigated.